pkg/test/framework/suite.go

```diff
@@ -117,7 +117,7 @@ func (s *Suite) RequireEnvironmentVersion(version string) *Suite {
 			if err != nil {
 				return fmt.Errorf("failed to get Kubernetes version: %v", err)
 			}
-			if fmt.Sprintf("%s.%s", ver.Major, ver.Minor) >= version {
+			if fmt.Sprintf("%s.%s", ver.Major, ver.Minor) <= version {
```

Comment:
I think the Info message inside the block also needs to include the ver.Major.

Reply:
thanks, fixed